### ISSUE TITLE
Add Widget Source Code

### ISSUE THREAD

**Issue by innerjoin** [jazz-community/rtc-absence-widget#1]
This repository should also contain the source code of the project, not only the downloadable artifact. It's an important aspect of open source software that the source is open to the public, otherwise it would just be called freeware

**Comment by chinmayeearadhya** [comment]
I agree, Thank you for at least providing the downloadable artifact; that's already helpful. 
However, having access to the source would allow us to make minor modifications, like changing the widget's color, and contribute improvements more effectively. Hope to see the source code shared soon!
